### DISCUSSION TITLE
Team meeting planning discussion 2022-16-02

### DISCUSSION THREAD

**Discussion by nalininsbs** [wpmarketingteam/Marcomms-Internal-Comms#14] [General]
This is to start for next week's meeting.

Important to promote the marketing team celebration, thank you to the previous team representatives and welcome to the new ones. This also needs promoting during the week.

Will there be dates in February about the next major release?

**Comment by murillol** [comment]
Hi! Do we plan the meeting directly in this discussion, or is there also a Google Doc for planning?

**Reply by nalininsbs** [reply]
We had a script document last week. I can not find the folder where this is saved. 
It is very late for me now so I am copying the draft of the agenda I started below @murillol @eidolonnight . If you want to add to it. Thank you.

1. Welcome
2.i) Participants live and async.
ii) Welcoming new joiners and returners to the channel
ii) Information on how to contribute to the team
iii) Social question of the week

3. Notes from the Last Meeting
https://make.wordpress.org/marketing/2022/02/14/notes-global-marketing-meeting-09-february-2022/
Volunteer call for notetakers.

4. Project Updates

5. WordPress Marketing Team Celebration
This will be on 17 February 2022 at 15:00 UTC. Let the team know if you will be able to join. A link to the video link will be shared in the channel before the event.
Join us at the WordPress Marketing Team Celebration for outgoing and incoming team reps, as well as all the contributors that have contributed to the Make Marketing Team since its creation in 2014!

7. Diary/Calendar Dates

8. Open Floor

**Reply by nalininsbs** [reply]
The draft on the blog https://make.wordpress.org/marketing/?p=8686&preview=1&_ppp=e78b4a264b

@murillol and @eidolonnight if there are any updates could you add and review the post? Do publish it when you are ready in your timezone. Thank you.

**Reply by nalininsbs** [reply]
Social question of the week: when did you discover WordPress?

**Reply by eidolonnight** [reply]
Updates made and post published.

**Comment by murillol** [comment]
- Thanks @nalininsbs for sharing the draft!
- Thanks @yvettesonneveld for sharing the link to the folder where the Script docs are: https://drive.google.com/drive/u/2/folders/1-bzerICA7Hj6cqslEIwJX0cgtR9Otdo2
- Point 3 of the Agenda, we could put **Social Media** update, **Learn WP**,  new **design of /news**,  and the upcoming **WP Diversity** event? @eidolonnight

**Reply by murillol** [reply]
Link to Meeting Script: https://docs.google.com/document/d/1AhuFm25ZfZwyywkA4dgzfNVrdNAZ8sETB7u4nKiDGlc/edit?usp=sharing

**Comment by murillol** [comment]
@nalininsbs @eidolonnight,  I've left [a draft of the script](https://docs.google.com/document/d/1AhuFm25ZfZwyywkA4dgzfNVrdNAZ8sETB7u4nKiDGlc/edit#), and I would suggest slightly updating the Agenda items to match on the P2:


1. Welcome

2a. Attendance and New Contributors
2b. Social Question

3. Notes and Actions

4. Project Updates
4a. Monthly Meetup Organizer Newsletter
4b. Social Media
4c. Learn WP
4d. GitHub (if there's anything new)
4e. WPDiversity
4f. News redesign

5. WordPress Marketing Team Celebration

6. Diary Dates

7. Open Floor

---
I have a work meeting on Wednesdays at 8.30am which ends right before our WP meeting starts. I will try to be online about 5 minutes before, but if any changes are to be made to the script or P2, please go ahead because I won't be able to give any input tomorrow morning.
